src/Nox.Types.EntityFramework/Abstractions/NoxDatabaseConfigurator.cs

```diff
@@ -107,8 +113,43 @@ public abstract class NoxDatabaseConfigurator : INoxDatabaseConfigurator
             }
         }
 
-        private void ConfigureRelationForeignKeyProperty(NoxSolutionCodeGeneratorState codeGeneratorState,
-            EntityTypeBuilder builder,
+        public virtual void ConfigureOwnedRelationships(
+            NoxSolutionCodeGeneratorState codeGeneratorState,
+            IEntityBuilderAdapter builder,
+            Entity entity,
+            IReadOnlyList<EntityRelationshipWithType> ownedRelationshipsToCreate,
+            NoxSolution noxSolution,
```

Comment:
the codeGeneratorState already contains a Solution no need to input it again

Reply:
Done, nice catch